app/Notifications/PasswordResetNotification.php

```
@@ -0,0 +1,51 @@
+    use Queueable;
+
+    public function __construct(
+        private readonly string $token
+    ) {}
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @return array<int, string>
+     */
+    public function via(User $notifiable): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$notifiable'. (undefined)

(UnusedFormalParameter)